SmartDeviceLinkTests/UtilitiesSpecs/HTTP Connection/SDLURLRequestTaskSpec.m

```diff
@@ -1,140 +0,0 @@
-#import <Quick/Quick.h>
-#import <Nimble/Nimble.h>
-#import <OHHTTPStubs/OHHTTPStubs.h>
-
-#import "SDLURLRequestTask.h"
-
-
-QuickSpecBegin(SDLURLRequestTaskSpec)
```

Comment:
We shouldn't need to do this, as we are using them for different URL request types (POST, GET). Do we really need to test to make sure that an Apple-built framework works?